### PR TITLE
Adding MySQL 8.0 support

### PR DESCRIPTION
**Pull request by imriss**
Adding MySQL 8.0 support

### CODE REVIEW

File: pkg/internal/mysqlcluster/mysqlcluster.go [hunk 1]
```diff
@@ -200,6 +200,22 @@ func (c *MysqlCluster) GetMysqlImage() string {
 	return ""
 }
 
+// GetMysqlInitImage returns the mysql image for current mysql cluster
+func (c *MysqlCluster) GetMysqlInitImage() string {
```
[the remark this answers is not in the document]
Comment (imriss) [reply pkg/internal/mysqlcluster/mysqlcluster.go]: It was part of a test to resolve a missing table issue. The test was reverted later on. I will remove this function.

**Comment by AMecea** [comment]
Hi @imriss, can you please rebase this from master, this will fix the CI step that fails? I would like to merge it. :smile:

**Comment by imriss** [comment]
Hi @AMecea, please check :)

**Comment by AMecea** [comment]
I've tested this manually after merge but does not work. I will open a PR with some fixes.

This brakes backward compatibility with mysql 5.7 and also mysql 8.0 does not start with current value for `sql_mode`.

**Pull request merged**